### DISCUSSION TITLE
Date for version 1.0???

### DISCUSSION THREAD

**Discussion by H1Mohamed** [pocketbase/pocketbase#4036] [Q&A]
its been a long time since this project has started and its taking a long time to get to version 0.21 , its kinda concerning even tho the project has so much potential..so is there is a desired date for v1.0 or is it just based on the time it takes to make versions???

**Comment by ganigeorgiev** [comment]
There are no ETAs.
This is a hobby project and I don't want to make promises.
You shouldn't expect anything more than what is already available. See also https://github.com/pocketbase/pocketbase/discussions/3271

**Reply by saikoml** [reply]
Without v1 date is very difficult to convince my team to use PocketBase. Supabase is better in this regard and it is better documented. Shame...

**Reply by ganigeorgiev** [reply]
Supabase is a business. It is maintained by multiple full-time developers, it is well funded, has plenty of features, has commercial offering and enterprise support.

PocketBase is non of the above and there is nothing wrong if you prefer using Supabase.

I could understand your disappointment but you also have to keep in mind that this is a hobby project and I have my own personal life and things to take care for and at the end of the day if you are not satisfied with the pace and the direction where the project is going on you can always fork it or move on to something else.
While donations help covering some of my utility bills (_as of now combined from both OpenCollective and PayPal the monthly recurring amount is ~285BGN total_) there are far from enough to sustain myself and for the last ~1 year I'm literally burning through my savings so that I can continue working on my open source projects.

As mentioned in https://github.com/pocketbase/pocketbase/discussions/3271, after the Presentator v3 release, I'll try to outline some of the next short term plans for PocketBase but I don't want to make promises or feel obligated to ETAs (_I made that mistake with Presentator and every 1-2 weeks there is someone asking when it will be ready_).

**Comment by philipthomasme** [comment]
I just wanted to use this thread with such negativity to say that this is a phenomenal project.
Incredible work - I love to use it.
Much respect and keep up the good work, @ganigeorgiev !
